docs/source/examples/cdr_qrack.md

```diff
@@ -0,0 +1,178 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.3
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# CDR with Qrack as Near-Clifford Simulator
+
+In this tutorial, [Clifford Data Regression](../guide/cdr.md) (CDR) is used with [Qrack](https://qrack.readthedocs.io/en/latest/) and a Qiskit fake backend.
+
++++
+
+## Setup
+
+To start, relevant modules and libraries are imported. Please ensure that the following Python modules are installed: `mitiq`, `numpy`, `pyqrack`, `cirq`, `qiskit`
+
++++
+
+```{note}
+In the code below the environmental variable, `QRACK_MAX_CPU_QB`, is set to `-1`. This enviroment variable sets the maximum on how many qubits can be allocated on a single QEngineCPU instance. More information can be found on the [Qrack README page](https://github.com/unitaryfund/qrack?tab=readme-ov-file#maximum-allocation-guard).
+```
+
+```{code-cell}
+import numpy as np
+import collections
+
+import os
+import warnings
+warnings.simplefilter("ignore", np.ComplexWarning)
+
+from pyqrack import QrackSimulator, QrackCircuit
+os.environ["QRACK_MAX_CPU_QB"]="-1"
+
+import mitiq.interface.mitiq_qiskit
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+from mitiq import cdr, Observable, PauliString
+
+import cirq
+
+from qiskit.providers.fake_provider import Fake5QV1
+```
+
+## Sample Circuit
+
+This sample circuit includes Clifford gates (`H`, `CNOT`, `RX`) and non-Clifford gates (`RZ`).
+
+```{code-cell}
+a, b = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(
+    cirq.H.on(a),  # Clifford
+    cirq.H.on(b),  # Clifford
+    cirq.rz(1.75).on(a),
+    cirq.rz(2.31).on(b),
+    cirq.CNOT.on(a, b),  # Clifford
+    cirq.rz(-1.17).on(b),
+    cirq.rz(3.23).on(a),
+    cirq.rx(np.pi / 2).on(a),  # Clifford
+    cirq.rx(np.pi / 2).on(b),  # Clifford
+)
+
+# CDR works better if the circuit is not too short. So we increase its depth.
+circuit = 5 * circuit
+print(circuit)
+```
+
+## Qrack Near-Clifford Simulator
```

Comment:
Right before we jump into defining the three executors, can we, in a few sentences, let readers know why we need three different executors. Otherwise it may be confusing when after defining each executor, another is defined.

Reply:
I added a header section for the executors along with an explanation. Let me know if it needs more.